docs/en/query_language/functions/bit_functions.md

```diff
@@ -204,5 +204,40 @@ Result:
 │                    0 │
 └──────────────────────┘
 ```
+## bitCount {#bitcount}
+
+Calculates the number of bits set to one in a number.
+
+**Syntax** 
+
+```sql
+bitCount(x)
+```
+
+**Parameters** 
+
+- `x` — [Integer](../../data_types/int_uint.md) or [floating point](../../data_types/float.md) number.
+
+**Returned value**
+
+- Number of bits set to one in the input number.
+
+Type: [Integer](../../data_types/int_uint.md).
+
+**Example**
+
+Query:
+
+```sql
+SELECT bitCount(100)
```

Comment:
we'd better use a number that can't be confused if its binary or decimal + add a comment showing how it looks like in binary